core/azure-core/src/main/java/com/azure/core/implementation/annotation/ServiceInterface.java

```diff
@@ -14,9 +14,9 @@
  */
 @Target({TYPE})
 @Retention(RetentionPolicy.RUNTIME)
-public @interface Service {
+public @interface ServiceInterface {
     /**
-     * Name of the service.
+     * Name of the service - this must be short and without spaces.
      * @return the service name given to the interface.
      */
     String value() default "";
```

Comment:
Should this be made a required field instead of having a default empty string?

Reply:
I agree